GameData/RealismOverhaul/RO_SAFConfig.cfg

```diff
@@ -0,0 +1,90 @@
+/// Example config - Replace 'MyPartWithSAFModule' with your part name
+/// @PART[MyPartWithSAFModule]:FOR[RealismOverhaul]
+/// {
+/// 	@MODULE[ModuleSimpleAdjustableFairing]
+/// 	{
+/// 		ROSAFRescale = 1.81 //Make everything 1.81 times bigger
+/// 	}
+/// }
+
+@PART[*]:HAS[@MODULE[ModuleSimpleAdjustableFairing]:HAS[#ROSAFRescale]]:AFTER[RealismOverhaul]
+{
+	@MODULE[ModuleSimpleAdjustableFairing]
+	{
+		&scale = 1
+		@scale *= #$ROSAFRescale$
+		@segmentLength *= #$ROSAFRescale$
+		@shieldingCenter[*] *= #$ROSAFRescale$
+		@shieldingBaseRadius *= #$ROSAFRescale$
+		@editorOpenOffset[*] *= #$ROSAFRescale$
+
+		@WALL_BASE
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@WALL
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@CONE
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@CAP
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+	}
+
+	@MODULE[ModuleB9PartSwitch]:HAS[#moduleID[fairingSwitch]]
+	{
+		@SUBTYPE,*
+		{
+			@MODULE:HAS[@IDENTIFIER[ModuleSimpleAdjustableFairing]]
+			{
+				@DATA
+				{
+					@WALL_BASE
+					{
+						@CoM[*] *= #$../../../../../MODULE[ModuleSimpleAdjustableFairing]/ROSAFRescale$
```

Comment:
`/MODULE[...]/ROSAFRescale`

(`/` starts from the root of the part, `@` starts from the top-level. No need for `../` spam.)

Reply:
...hmm that might catch the wrong MODULE if there are multiple. Not sure if `:HAS` is allowed in a path off the top of my head.

Reply:
Supporting multiple modules seems difficult, might leave that for when it's needed.

Reply:
Made the patch apply on all SAF modules and all B9PS modules